docs/google-cloud-gke/access-result.md

```diff
@@ -3,6 +3,6 @@ The results produced by the surge process are stored in the Google Cloud Storage
 1. Install `gsutil` by following the documentation [here](https://cloud.google.com/storage/docs/gsutil_install).
 2. To access the bucket, click on "Cloud Storage" >> "Buckets." Choose the "surge-results" bucket and verify the existence of the created folder. Next, click the "DOWNLOAD" button at the top and execute the gsutil command locally to initiate the data download.
 
-<img  src="/gke/access-result.1.png" alt="Access the result" style="width: 100vw">
-<img  src="/gke/access-result.2.png" alt="Access the result" style="width: 100vw">
-<img  src="/gke/access-result.3.png" alt="Access the result" style="width: 100vw">
\ No newline at end of file
+<img  src="/docs/public/gke/access-result.1.png" alt="Access the result" style="width: 100vw">
```

Comment:
All the images that are moved to the public folder are broken. Please correct the image path to e.g. here it would be "/public/gke/access-result.1.png".

Reply:
In docs/google-cloud-gke/ folder, the image links were broken in all the md files. That is why it was changed to /docs/public/gke/. 

Now, I checked again from github the preview. When it is /public/gke/access-result.1.png the image does not appear, so broken; and it works only when /docs/  is added like /docs/public/gke/access-result.1.png. If there is something I miss, please correct me.

Reply:
Thanks for the feedback, I checked the md files only from github preview not with npm run. That is where it is broken.

I ran the npm docs and checked from there. All the links are changed to /public/gke/.. now in all the md files, checked and it works there. I committed the changes.

Reply:
@NishaSharma14 yesterday in my pull request, I removed the empty md files from the docs folder. Do we need them ?

Reply:
Thanks Aziz